service/ntfy/ntfy.go

```diff
@@ -0,0 +1,174 @@
+package ntfy
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/pkg/errors"
+)
+
+// Service allow you to configure Ntfy service.
+type Service struct {
+	client     *http.Client
+	serverURLs []string
+}
+
+func defaultHTTPClient() *http.Client {
+	return &http.Client{
+		Timeout: 5 * time.Second,
+	}
+}
+
+// DefaultServerURL is the default server to use for the Ntfy service.
+const DefaultServerURL = "https://ntfy.sh"
+
+// normalizeServerURL normalizes the server URL. It prefixes it with https:// if it's not already and appends a slash
+// if it's not already there. If the serverURL is empty, the DefaultServerURL is used. We're not validating the url here
+// on purpose, we leave that to the http client.
+func normalizeServerURL(serverURL string) string {
+	if serverURL == "" {
+		return DefaultServerURL
+	}
+
+	// Normalize the url
+	if !strings.HasPrefix(serverURL, "http") {
+		serverURL = "https://" + serverURL
+	}
+	if !strings.HasSuffix(serverURL, "/") {
+		serverURL = serverURL + "/"
+	}
+
+	return serverURL
+}
+
+// AddReceivers adds server URLs to the list of servers to use for sending messages.
+func (s *Service) AddReceivers(serverURLs ...string) {
+	for _, serverURL := range serverURLs {
+		serverURL = normalizeServerURL(serverURL)
+		s.serverURLs = append(s.serverURLs, serverURL)
+	}
+}
+
+// NewWithServers returns a new instance of Ntfy service. You can use this service to send messages to Ntfy. You can
+// specify the servers to send the messages to. By default, the service will use the default server
+// (https://api.day.app/) if you don't specify any servers.
+func NewWithServers(serverURLs ...string) *Service {
+	s := &Service{
+		client: defaultHTTPClient(),
+	}
+
+	if len(serverURLs) == 0 {
+		serverURLs = append(serverURLs, DefaultServerURL)
+	}
```

Comment:
Length check is obsolete. An empty list will not alter the existsing serverURLs anyway.
```suggestion
	serverURLs = append(serverURLs, DefaultServerURL)	
```